api/v1beta1/common.go

```diff
@@ -109,3 +110,87 @@ func (annotations *BlockchainAnnotationList) DeleteAnnotation(k string) error {
 	annotations.LastDeleteTime = time.Now().String()
 	return nil
 }
+
+func (annotation *BlockchainAnnotation) GetID(k string) (ID, bool) {
+	if annotation.IDs == nil {
+		annotation.IDs = make(map[string]ID)
+	}
+	id, ok := annotation.IDs[k]
+	return id, ok
```

Comment:
I think it can be written like this.
```go
return annotation.IDs[k]
```

Reply:
`ok` is returned to indicate whether or not this id exists. Since  there will be many existence checks (ca enroll,add peer,add orderer,etc..),better to give this flag in its returns.

Reply:
👌